addons/purchase/report/purchase_report.py

```diff
@@ -7,7 +7,7 @@
 
 import re
 
-from odoo import api, fields, models, tools
+from odoo import api, fields, models, tools, _
```

Comment:
```suggestion
from odoo import api, fields, models, _
```
![Selection_675](https://user-images.githubusercontent.com/25005517/204252564-87977d40-d58d-4eb1-88ae-94837b09fd4b.png)

Reply:
thanks @MiquelRForgeFlow , updated